ddtrace/contrib/django/utils.py

```diff
@@ -121,16 +120,9 @@ def get_request_uri(request):
                 return None
         urlparts[k] = v
 
-    # DEV: With PY3 urlunparse calls urllib.parse._coerce_args which uses the
-    # type of the scheme to check the type to expect from all url parts, raising
-    # a TypeError otherwise. If the scheme is not a str, the function returns
-    # the url parts bytes decoded along with a function to encode the result of
-    # combining the url parts. We returns a byte string when all url parts are
-    # byte strings.
-    # https://github.com/python/cpython/blob/02d126aa09d96d03dcf9c5b51c858ce5ef386601/Lib/urllib/parse.py#L111-L125
-    if PY3 and not all(isinstance(value, binary_type) or value is None for value in urlparts.values()):
+    if PY3:
         for (key, value) in urlparts.items():
             if value is not None and isinstance(value, binary_type):
                 urlparts[key] = to_unicode(value)
 
-    return parse.urlunparse(parse.ParseResult(**urlparts))
+    return "".join((urlparts["scheme"], "://", urlparts["netloc"], urlparts["path"]))
```

Comment:
I'm wary of moving away from the library function (but super happy to see the above stuff go away). What are the performance differences we're talking about here?

Reply:
`get_request_uri` is about 9% of the total overhead. This change reduces that down to 3.5%